packages/terra-site/src/examples/date-picker/DatePickerDefaultDate.jsx

```diff
@@ -2,11 +2,29 @@ import React from 'react';
 import moment from 'moment';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-default-date"
-    selectedDate={moment().format()}
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
+        <DatePicker
+          name="date-input-default-date"
+          selectedDate={moment().format()}
+          onChange={this.handleDateChange}
+        />
+      </div>
+    );
+  }
+}
```

Comment:
This example renders a blank ISO on first load although it has a valid selected date